.pre-commit-config.yaml

```diff
@@ -45,8 +45,8 @@ repos:
       - id: flake8
 
   # Black for auto code formatting
-  - repo: https://github.com/psf/black
-    rev: 22.12.0
-    hooks:
-     - id: black
-       language_version: python3.8
+#  - repo: https://github.com/psf/black
+#    rev: 22.12.0
+#    hooks:
+#     - id: black
+#       language_version: python3.8
```

Comment:
Disabling black to prevent merge conflicts with master

Reply:
we can run black once the big overhaul is done.  you can also add a `skip: ()` element to the config file but this is totally fine.

Reply:
actually i think the skip might be for CI only. so this is good to just comment out fully until we are done with this big overhaul!